### ISSUE TITLE
Fix example knowledgetree/git_scan

### ISSUE THREAD

**Issue by timurgordon** [crystaluniverse/crystallib#274]
Example doesn't work:

![telegram-cloud-photo-size-4-5884098203769487015-y](https://github.com/freeflowuniverse/crystallib/assets/31495328/0ac5608a-a2f4-4b5e-879e-9c7632a3bd30)

**Comment by timurgordon** [comment]
Issue occurs when parsing link `![alt_text](img/internet_distance.png )` in the code

```
if link.url.contains(' ') {
	// to support something like
	//![](./img/license_threefoldfzc.png ':size=800x900')
	splitted := link.url.trim(' ').split(' ')
	link.filename = splitted[0]
		
	link.extra = splitted[1]
} 
```

**Comment by timurgordon** [comment]
fixed with 52302e63c2d3bae10382c8540576bbf582e9a18b
